packages/blocks/src/_common/components/hover/controller.ts

```diff
@@ -105,7 +105,8 @@ export class HoverController implements ReactiveController {
         return;
       }
 
-      this._abortController?.abort();
+      if (this._abortController) return;
```

Comment:
This line of code change will sometimes make the tooltip unable to display. You can see the tooltip for `underline` 
is no longer showing.

https://github.com/toeverything/blocksuite/assets/20479050/fab68156-6dcf-40f7-8517-a8ee862d542c

Reply:
> Cloud you let me know what happens occur if the abortController abort?
It is the abortController for 
https://github.com/toeverything/blocksuite/blob/c3dda83b8dc5ac74527cef3899e7754be1bdcb49/packages/blocks/src/_common/components/portal.ts#L120
Which on abort will remove the portal. 
The hover controller renders the `template` into the `portal`

Reply:
> This line of code change will sometimes make the tooltip unable to display. You can see the tooltip for underline
is no longer showing.

Both the List and More menus utilize the `HoverController`. For example, when you click the More button, it opens the More menu portal. However, when you move the mouse away from the button, the whenHover function is triggered with hover = false. At this point, if `abortController.abort()` is called, it will close the portal, causing issues with the code block. The hover controller might be removed based on https://github.com/toeverything/blocksuite/pull/7248, but the existing logic should be maintained.

The problem arises because abortController.abort() is not being triggered in the transition end event due to rapid mouse movements, which should set the abortController to undefined. Removing the transition may resolve this issue.